octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -124,8 +141,18 @@ def octavia(
         click.echo(click.style("🐙 - Project is not yet initialized.", fg="red", bold=True))
 
 
-def get_api_client(airbyte_url: str, user_agent: str, api_http_headers: Optional[List[ApiHttpHeader]]):
-    client_configuration = airbyte_api_client.Configuration(host=f"{airbyte_url}/api")
+def get_api_client(
+    airbyte_url: str, airbyte_username: str, airbyte_password: str, user_agent: str, api_http_headers: Optional[List[ApiHttpHeader]]
+):
+    # basic_auth = ""
+    # if len(airbyte_username) > 0:
+    #     basic_auth += airbyte_username
+    # if len(airbyte_password) > 0:
+    #     if len(airbyte_username) > 0:
+    #         basic_auth += ":"
+    #     basic_auth += airbyte_password
+
+    client_configuration = airbyte_api_client.Configuration(host=f"{airbyte_url}/api", username=airbyte_username, password=airbyte_password)
```

Comment:
@alafanechere I'm giving a try to updating the CLI.  It seems that `airbyte_api_client.Configuration` accepts username and password, but the url libraray dosen't seem to get it:

`pytest --cov=octavia_cli`

```
self = <airbyte_api_client.api_client.Endpoint object at 0x106041610>
kwargs = {'_check_input_type': True, '_check_return_type': True, '_content_type': None, '_host_index': None, ...}, index = None, server_variables = {}
_host = 'http://localhost:8000/api', key = '_host_index', value = None, params = {'body': None, 'collection_format': {}, 'file': {}, 'form': [], ...}
accept_headers_list = ['application/json']
```

Any tips?

Reply:
I guess the problem is indeed on the generated `ApiClient` class, I think it is not setting the headers according to the `Configuration` it is built with.
I found a workaround in my latest commit and checked its working with an addition to the existing `test_api_http_headers` integration tests.